app/src/main/java/com/example/scoutconcordia/FileAccess/DES.java

```diff
@@ -1,4 +1,4 @@
-package com.example.scoutconcordia;
+package FileAccess;
```

Comment:
should be: package com.example.scoutconcordia.FileAccess;

Reply:
Fixed